src/GradhSph/GradhSphTree.cpp

```diff
@@ -266,13 +247,6 @@ void GradhSphTree<ndim,ParticleType>::UpdateAllSphProperties
     //=============================================================================================
 
     // Free-up all memory
-    delete[] r;
```

Comment:
I'm guessing the memory allocated in the ngb/ngb2 vectors are automatically deallocated when their destructor is called? So no need to manually deallocate them here?